fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1186,6 +1186,7 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			options.sync = !SERVER_KNOBS->ROCKSDB_UNSAFE_AUTO_FSYNC;
 			if (SERVER_KNOBS->ROCKSDB_DISABLE_WAL_EXPERIMENTAL) {
 				options.disableWAL = true;
+				options.sync = false;
```

Comment:
The code throws error if we don't set the sync to false when wal is enabled.
I was in confused in my last commit and removed it by mistake.